docs/standardized-schedule-method-for-initial-margin-calculation.md

```diff
@@ -0,0 +1,225 @@
+
+The **Standard Initial Model** (SIMM), developed by ISDA, offers a risk-sensitive, standardized IM calculation reflecting netting and diversification benefits, making it suitable larger institutions with complex portfolios. In contrast, the **Standardized Schedule** provides a simpler, conservative calculation method facilitating compliance for smaller firms or those without advanced modeling capabilities. Both approaches helps institutions meet BCBS-IOSCO guidelines, balancing regulatory compliance with effective risk management.
+
+## Standardized Schedule Method
+
+The **standardized schedule method** offers a simplified and **less risk-sensitive approach** to calculating Initial Margin (IM). It is particularly designed for market participants who may not possess the expertise or resources to develop and maintain sophisticated quantitative models.
+
+By utilizing this method, market participants can ensure that margin calculations are straightforward and accessible. This is crucial for smaller firms or those entering the market, as it levels the playing field. This method provides a **conservative approach**, ensuring that initial margin calculations adequately cover potential exposures without overly complicating the process.
+
+The **standardized schedule** is aligned with **regulatory requirements**, promoting adherence to **BCBS-IOSCO guidelines**. This alignment helps institutions avoid penalties and enhance their overall compliance posture.
+
+## Key components of the Standardised Schedule
+
+The standardized schedule defines specific **margin rates** for various asset classes, durations, and notional exposures. This provides a clear framework for determining the necessary initial margin amounts.
+
+| Asset Class            | Margin Rate |
+| ---------------------- | ----------- |
+| Credit: 0-2 year duration    | 2%          |
+| Credit: 2-5 year duration    | 5%          |
+| Credit: 5+ year duration    | 10%          |
+| Commodity    | 15%          |
+| Equity | 15%         |
+| Foreign exchange     | 6%         |
+| Interest rate: 0-2 year duration     | 1%         |
+| Interest rate: 2-5 year duration     | 2%         |
+| Interest rate: 5+ year duration     | 4%         |
+| Other     | 15%         |
+
+For example, credit derivatives generally have higher margin rates due to their increased default risk, while interest rate derivatives have lower rates due to their relatively stable nature.
+
+Each asset class is assigned a predetermined margin rate based on its inherent risk characteristics, duration, and exposure levels. This differentiation ensures that margin requirements are proportional to the risks associated with each asset type.
+
+## Standardized Initial Margin Formula
+
+The formula for calculating net standardized initial margin is as follows:
+     ```
```

Comment:
can you please replace this line with 
``` MD

added MD at the end so that this section of code is ignored in tests as its not anything related code written in .rosetta file types, functions, rules etc